srcpkgs/linux-wifi-hotspot/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'linux-wifi-hotspot'
+pkgname=linux-wifi-hotspot
+version=4.5.0
+revision=1
+archs="i686 x86_64 x86_64-musl"
```

Comment:
This probably shouldn't restrict the architectures, what is the reason?

Reply:
The CI failed for the other architectures, I think it was something that an X11 C library wasn't found